clang/test/CodeGen/fminimum-num-autovec.c

```diff
@@ -0,0 +1,407 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Should not be testing the vectorizer in clang. This needs a pure IR test in the backend

Reply:
We have the backend test already.
This PR fixes the front to generate vectorized IR from C code.

Reply:
You don't have a backend vectorization test. This PR contains no clang changes. You should delete the clang test and replace it with a backend IR test